roles/edpm_ovn/tasks/configure.yml

```diff
@@ -14,6 +14,18 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Apply ovsdb configuration from configMaps
+  block:
+  - name: Load ovsdb configuration from file
+    ansible.builtin.include_vars:
+      file: "{{ edpm_ovn_config_src }}/ovsdb-config"
+      name: edpm_ovn_configmap_facts
+    delegate_to: localhost
+
+  - name: Append ovsdb configMap options to external_ids
+    ansible.builtin.set_fact:
+      edpm_ovn_ovs_external_ids: "{{ edpm_ovn_ovs_external_ids | combine(edpm_ovn_configmap_facts) }}"
```

Comment:
if above task is changed then can adjust this to combine(edpm_ovn_configmap_facts) -> combine(edpm_ovn_configmap_facts|default({}))